cmd/posttriage/checks.go

```diff
@@ -13,6 +14,12 @@ import (
 type triageCheck func(jira.Issue) (triaged bool, msg string, err error)
 
 func priorityCheck(issue jira.Issue) (bool, string, error) {
+	// If a bug has been closed as a non-bug, we shouldn't insist on a priority.
+	// Taken from https://issues.redhat.com/rest/api/2/resolution
+	nonBugs := []string{"Won't Do", "Cannot Reproduce", "Can't Do", "Duplicate", "Not a bug", "Obsolete"}
+	if slices.Contains(nonBugs, issue.Fields.Resolution.Name) {
```

Comment:
nit: you could do that without being more verbose and without allocating slices:

```Go
	switch issue.Fields.Resolution.Name {
	case "Won't Do", "Cannot Reproduce", "Can't Do", "Duplicate", "Not a bug", "Obsolete":
		return true, "", nil
	}
```

Reply:
Forgot that switch statements are a thing in Go :sweat_smile: Done here https://github.com/shiftstack/bugwatcher/pull/81